homeassistant/components/ohme/quality_scale.yaml

```diff
@@ -0,0 +1,90 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration has no custom actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration has no custom actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      This integration has no explicit subscriptions to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      This integration has no custom actions and read-only platform only.
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      This integration has no options flow.
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: todo
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: todo
+  discovery:
+    status: exempt
+    comment: |
+      All supported devices are cloud connected. Discovery is not possible.
+  discovery-update-info:
+    status: exempt
+    comment: |
+      All supported devices are cloud connected. Discovery is not possible.
+  docs-data-update: todo
+  docs-examples: todo
+  docs-known-limitations: todo
+  docs-supported-devices: done
+  docs-supported-functions: todo
+  docs-troubleshooting: todo
+  docs-use-cases: todo
+  dynamic-devices:
+    status: exempt
+    comment: |
+      This integration only provides one device per config entry.
```

Comment:
No?

Reply:
Should I set these two to todo? Out of interest how would you pass both with an integration that like this with one account, one device?

Reply:
Oh i haven't read the other discussion. But you mean that you can only have one device in total? How did you come to that statement?

Reply:
In the vendors app there is no option to link multiple chargers, only to unlink the existing charger or replace the current charger with another. As a result it's max one charger per account. As a residential product I kind of understand why they haven't bothered

Reply:
How do their API look lik

Reply:
It does return a list for chargeDevices, so I guess could support multiple devices in future, the API library just doesn't have any support for this yet